game_clean/tc2/resource/ui/classloadoutpanel.res

```diff
@@ -213,58 +213,6 @@
 			"spotlight" "1"
 		
 			"modelname"		""
-			
```

Comment:
@wgetJane Removal of this might cause that stand issue, but this was most definitely removed for a reason in tf2 hud fix, so best is to ask someone who is hud expert

Reply:
Should be fixed now